code/game/objects/items/devices/paicard.dm

```diff
@@ -92,10 +92,26 @@
 					to_chat(pai, "<span class='userdanger'>Your mental faculties leave you.</span>")
 					to_chat(pai, "<span class='rose'>oblivion... </span>")
 					qdel(pai)
-		if(href_list["wires"])
-			var/wire = text2num(href_list["wires"])

```

Comment:
Seems to me the only issue here was text2num since wire defines are text. Proper fix here would be changing these to simple preset and switch both so it works and you can't cut wires other than these two.

Reply:
@AnturK I tried what you have here and it didn't work. It goes back to the original state where the Radio Uplink options never update from "Enabled"

Reply:
That doesn't make much sense, Were the wires actually cut ? Why does this even use wires instead toggling listening/broadcasting on the radio anyway.

Reply:
@AnturK pAI's currently have full access to their radio settings. This seeks to override that so that humans can silence/deafen pAI's and without the pAI just reverting it.

Reply:
hmm... if people are being a nuisance as pai couldn't you just delete them?

Reply:
Ohh right, i completely missed the fact this is not pai interface.

Reply:
@ExcessiveUseOfCobblestone Yeah kinda, but I was thinking of doing a PR soon where EMP's just knocked out the holoform and radio settings, requiring a human to provide the pAI with helpies.